services/brig/src/Brig/API/User.hs

```diff
@@ -402,6 +404,11 @@ createUser new = do
           return Nothing
       pure pdata
 
+initAccountFeatureConfig :: UserId -> AppIO ()
+initAccountFeatureConfig uid = do
+  mbCciDefNew <- asks (^. settings . getAfcConferenceCallingDefNewMaybe)
```

Comment:
You can just use `view` with a lens instead of `ask . view`.

Reply:
https://github.com/wireapp/wire-server/pull/1818/commits/77102286b3af65b9b7feb48d31e8f466a9b2f7ed

(thanks :)